newm-server/src/test/kotlin/io/newm/server/features/earnings/EarningsRoutesTests.kt

```diff
@@ -0,0 +1,169 @@
+package io.newm.server.features.earnings
+
+import com.google.common.truth.Truth
+import io.ktor.client.call.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import io.newm.server.BaseApplicationTests
+import io.newm.server.features.earnings.model.Earning
+import io.newm.server.features.earnings.model.GetEarningsResponse
+import io.newm.server.features.song.database.SongTable
+import io.newm.server.features.song.model.MintingStatus
+import io.newm.server.features.song.model.Release
+import io.newm.server.features.song.model.ReleaseType
+import io.newm.server.features.song.model.Song
+import io.newm.server.features.user.database.UserTable
+import io.newm.server.features.user.model.User
+import kotlinx.coroutines.runBlocking
+import org.jetbrains.exposed.sql.deleteAll
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+private const val EARNINGS_PATH = "v1/earnings"
+
+class EarningsRoutesTests : BaseApplicationTests() {
+    @BeforeAll
+    fun beforeAllEarningsRoutesTests() {
+    }
```

Comment:
You can remove this function, since it's empty

Reply:
I will take care of that :) 
I've a test failing, I couldn't figure out what is the root cause! compared it with other routing tests, but I didn't see a difference!

Reply:
@wlara  This new test is failing `EarningsRoutesTests` and I can't figure out what's missing in the setup! keep getting `404 Not Found`